packages/react-components/review/react-components.api.md

```diff
@@ -145,6 +161,106 @@ export interface JumpToNewMessageButtonProps {
 // @public
 export const lightTheme: PartialTheme & CallingTheme;
 
+// @public
+export const LocaleContext: React_2.Context<ILocale>;
+
+// @public (undocumented)
+export const locales: {
+    'en-US': {
+        locale: {
+            locale: string;
+            strings: {
```

Comment:
nit: could we change this 'strings' name to something like 'componentStrings' instead. I'm just afraid of confusion with the type 'string'

Reply:
That's good point.. Will do.